examples/jaicp-telephony/src/main/kotlin/com/justai/jaicf/examples/jaicptelephony/TelephonyBotModule.kt

```diff
@@ -1,12 +1,12 @@
-package com.justai.jaicf.examples.jaicpexamples
+package com.justai.jaicf.examples.jaicptelephony
 
 import com.justai.jaicf.channel.jaicp.channels.TelephonyEvents
 import com.justai.jaicf.channel.jaicp.dto.telephony
 import com.justai.jaicf.channel.jaicp.reactions.telephony
 import com.justai.jaicf.helpers.logging.WithLogger
 import com.justai.jaicf.model.scenario.Scenario
 
-class CitiesGameBotModule : Scenario(), WithLogger {
+class TelephonyBotModule : Scenario(), WithLogger {
```

Comment:
Why not to use `object` instead of class?

Reply:
Yep, thanks for noticing.